workflows/convert_input_format.py

```diff
@@ -0,0 +1,147 @@
+#     confusingly similar designation, without the U.S. Department of Energy's
+#     prior written consent.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+import os
+import platform
+import shutil
+import subprocess
+
+from eplaunch.workflows.base import BaseEPLaunchWorkflow1, EPLaunchWorkflowResponse1
+
+
+class ConvertInputFormatWorkflow(BaseEPLaunchWorkflow1):
+
+    def name(self):
+        return "ConvertInputFormat-${CMAKE_VERSION_MAJOR}.${CMAKE_VERSION_MINOR}.${CMAKE_VERSION_PATCH}"
+
+    def context(self):
+        return "EnergyPlus-${CMAKE_VERSION_MAJOR}.${CMAKE_VERSION_MINOR}.${CMAKE_VERSION_PATCH}-${CMAKE_VERSION_BUILD}"
+
+    def description(self):
+        return "Run ConvertInputFormat"
+
+    def get_file_types(self):
+        return ["*.idf", "*.epJSON"]
+
+    def get_output_suffixes(self):
+        return [".idf", ".epJSON"]
+
+    def get_extra_data(self):
+        return {"Hey, it's extra": "data"}
+
+    def get_interface_columns(self):
+        return []
+
+    def main(self, run_directory, file_name, args):
+        if 'workflow location' in args:
+            energyplus_root_folder, _ = os.path.split(args['workflow location'])
+            if platform.system() == 'Windows':
+                convertinputformat_binary = os.path.join(energyplus_root_folder, 'ConvertInputFormat.exe')
+            else:
+                convertinputformat_binary = os.path.join(energyplus_root_folder, 'ConvertInputFormat')
+            if not os.path.exists(convertinputformat_binary):
+                return EPLaunchWorkflowResponse1(
+                    success=False,
+                    message="ConvertInputFormat binary not found: {}!".format(coeffconv_binary),
+                    column_data=[]
+                )
+        else:
+            return EPLaunchWorkflowResponse1(
+                success=False,
+                message="Workflow location missing: {}!".format(args['worflow location']),
+                column_data=[]
+            )
+
+        original_with_path = os.path.join(run_directory, file_name)
+        converted_file_no_ext, original_ext = os.path.splitext(original_with_path)
+        if original_ext.lower() == '.idf':
```

Comment:
I actually think the `lower` is superfluous because the workflow won't run on other extensions.  And I don't think ConvertInputFormat will like other casing either.